MessagingService.BusinessLogic.Tests/DomainEventHanders/DomainEventHandlerResolverTests.cs

```diff
@@ -0,0 +1,101 @@
+﻿namespace MessagingService.BusinessLogic.Tests.DomainEventHanders
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using EmailMessage.DomainEvents;
+    using EventHandling;
+    using Moq;
+    using Shouldly;
+    using Testing;
+    using Xunit;
+
+    public class DomainEventHandlerResolverTests
+    {
+        [Fact]
+        public void DomainEventHandlerResolver_CanBeCreated_IsCreated()
+        {
+            Dictionary<String, String[]> eventHandlerConfiguration = new Dictionary<String, String[]>();
+
+            eventHandlerConfiguration.Add("TestEventType1", new String[] { "MessagingService.BusinessLogic.EventHandling.EmailDomainEventHandler" });
```

Comment:
Codacy found an issue: [Remove the array type; it is redundant.](https://app.codacy.com/manual/stuart_ferguson1/Messaging/pullRequest?prid=5953475)